sdk/cosmos/azure-cosmos-encryption/src/test/java/com/azure/cosmos/encryption/CosmosEncryptionAsyncClientCacheTest.java

```diff
@@ -0,0 +1,52 @@
+package com.azure.cosmos.encryption;
+
+import com.azure.core.cryptography.KeyEncryptionKeyResolver;
+import com.azure.cosmos.CosmosAsyncClient;
+import com.azure.cosmos.CosmosAsyncClientEncryptionKey;
+import com.azure.cosmos.CosmosAsyncContainer;
+import com.azure.cosmos.CosmosAsyncDatabase;
+import com.azure.cosmos.implementation.RequestOptions;
+import com.azure.cosmos.models.CosmosClientEncryptionKeyProperties;
+import com.azure.cosmos.models.CosmosClientEncryptionKeyResponse;
+import org.mockito.Mockito;
+import org.testng.annotations.Test;
+import reactor.core.publisher.Mono;
+
+
+public class CosmosEncryptionAsyncClientCacheTest {
+
+    @Test(groups = {"unit"}, timeOut = 40000)
+    public void clientEncryptionPropertiesAsync() {
```

Comment:
I just noticed we have [CosmosEncryptionClientCachesTest.java](https://github.com/Azure/azure-sdk-for-java/pull/27210/files#diff-48a3586de958f03be13321f60b950216ccfa9c87dd3482a5d6bb7cac3af8ac06) , please move it there